packages/addon/src/intent_resolver.ts

```diff
@@ -1,23 +1,27 @@
 import { Icon } from "@kite9/fdc3";
 import { AppIntent } from "@kite9/fdc3";
-import { ResolverIntents, ResolverMessageChoice, SingleAppIntent } from "@kite9/fdc3-common";
+import { IframeResolveActionPayload, IframeResolvePayload } from "@kite9/fdc3-common";
```

Comment:
Don't import the subtypes as their naming may be unstable. Please try and use the top-level types instead (and add comments about this).
Also needs updating to the FDC3UserInterface messages